DEPLOYMENT.md

```diff
@@ -2,6 +2,32 @@
 
 Host on Heroku. [Integrated with GitHub](https://devcenter.heroku.com/articles/github-integration).
 
+## Prepare your environment
+
+```
+cd your_projects_folder
+git clone https://github.com/rubytaiwan/rubytw-reboot.git
+heroku login
+heroku git:remote -a rubytw-staging -r staging
+heroku git:remote -a rubytw-production -r production
+git remote -v
+# you will got staging/production remote name now.
```

Comment:
I think we assume they already cloned, and this actually belongs to `bin/setup` script, could you remove this section and add following to `bin/setup` instead?

```ruby
# Set up deployments
git remote add staging git@heroku.com:rubytw-staging.git
heroku join --app rubytw-staging

git remote add production git@heroku.com:rubytw-production.git
heroku join --app rubytw-production
```